lib/XML/LibXML/Document.pm

```diff
@@ -9,24 +10,23 @@ use XML::LibXML::Attr;
 use XML::LibXML::Dom;
 use XML::LibXML::Enums;
 use XML::LibXML::Error;
+use XML::LibXML::Element;
 
-unit class XML::LibXML::Document is xmlDoc is repr('CStruct') does XML::LibXML::Nodish;
+multi trait_mod:<is>(Routine $r, :$aka!) is export { $r.package.^add_method($aka, $r) };
```

Comment:
I need to test this more, but when I tried to use it, it didn't seem to work, hence the copies in all of the modules. I will see if I can remove them now that everything is running with no problems and see if they are still needed.